ui/package.json

```diff
@@ -20,12 +20,14 @@
     "@angular/platform-browser": "^14.2.12",
     "@angular/platform-browser-dynamic": "^14.2.12",
     "@angular/router": "^14.2.12",
-    "moment": "^2.29.1",
+    "moment": "^2.29.4",
+    "moment-timezone": "^0.5.43",
     "ng-http-loader": "11.0.0",
     "ng-qrcode": "^7.0.0",
+    "ngx-cookie-service": "14.0.0",
     "ngx-countdown": "^12.0.1",
+    "ngx-timezones": "^1.0.6",
```

Comment:
ngx-timezones is not used in the project, so it should be removed from the package.json to avoid unnecesary dependencies

Reply:
Removed.